yt/visualization/plot_window.py

```diff
@@ -182,9 +182,19 @@ class PlotWindow(ImagePlotContainer):
         The size of the window on the longest axis (in units of inches),
         including the margins but not the colorbar.
     right_handed : boolean
+        Depreceated, please use flip_horizontal.
```

Comment:
I see that you're already issueing a deprecation warning when it's used (great !), but I think we should also raise an error if it's combined with its replacements.

Reply:
this one also slipped by, will add to the next batch.